### PR TITLE
Multi-stage dev/prod dockerization

### PR DESCRIPTION
**Pull request by int3l**
Lightweight alpine-based dev/prod docker files for convenience and small footprint.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+
+RUN go get -u github.com/golang/dep/cmd/dep && \
+    dep ensure && \
+    go build cmd/gortr/gortr.go
+
+FROM alpine:latest
+ARG src_dir
+
+RUN apk --update --no-cache add ca-certificates && \
+    adduser -S -D -H -h / rtr
```
Comment (int3l) [inline Dockerfile]: Maybe this line is redundant.

File: Dockerfile.prod [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ARG src_uri
+
+RUN apk --update --no-cache add git && \
+    go get -u $src_uri
+
+FROM alpine:latest
+ARG src_uri
+
+RUN apk --update --no-cache add ca-certificates && \
+    adduser -S -D -H -h / rtr
```
Comment (int3l) [inline Dockerfile.prod]: Maybe this line is redundant.

**Comment by lspgn** [comment]
Merged! Thank you @int3l :).

**Comment by int3l** [comment]
@lspgn thanks. I happy, because now gortr can be used in GNS3 and other topology creation tools as a standalone container :)

**Pull request merged**